### ISSUE TITLE
manual fixes once SRDB import is final

### ISSUE THREAD

**Issue by teixeirak** [forc-db/ForC#181]
Running list:

- [ ]  history/ plot name for Husnain_2014_ceft site "R-SF"

**Comment by beckybanbury** [comment]
There are a few instances where the species listed in veg.notes don't match with the dominant.veg classification. The description in the plot name often does match the dominant.veg classification, so we would have to decide if we want to override this or not @teixeirak @ValentineHerr . A running list of IDs in ForC to double check is:

- [ ] 20229 - 20270
- [ ] 20298 - 20327
- [ ] 20402 - 20429
- [ ] 26124 - 26134
- [ ] 26157 - 26167
- [ ] 28289
- [ ] 26901
- [ ] 25876 - 9
- [ ] 25465 - 8
- [ ] 21690 - 1

**Comment by beckybanbury** [comment]
@ValentineHerr @teixeirak I'm not clear from issue #183 how you want to deal with the differences in stand ages; Valentine, did you get any further with this at all?

In terms of correcting the notes, where are the "SRDB notes"? Is that in SRDB, or notes in the source.notes of ForC measurements?

**Comment by ValentineHerr** [comment]
I am working on issue 183

**Comment by teixeirak** [comment]
> In terms of correcting the notes, where are the "SRDB notes"? Is that in SRDB, or notes in the source.notes of ForC measurements?

They are in SRDB. I have created [this document](https://github.com/forc-db/ForC_private/blob/master/data%20to%20add/srdb/site_notes_to_review.csv) to identify and track the notes that need review. I started noting what needs to be done.

**Comment by beckybanbury** [comment]
> They are in SRDB. I have created [this document](https://github.com/forc-db/ForC_private/blob/master/data%20to%20add/srdb/site_notes_to_review.csv) to identify and track the notes that need review. I started noting what needs to be done.

So this is something that needs fixing in SRDB, not in ForC?

**Comment by teixeirak** [comment]
No, we need to go through the SRDB notes to capture relevant info that could not be dealt with by the automatic import. For example, notes about the geographic coordinates should go in a notes field in the SITES table, notes indicating different source for C variables other than respiration mean we should change the citation.ID in MEASUREMENTS, etc. 

Does my document make sense now?

**Comment by ValentineHerr** [comment]
update on issue #181,
I have fixed most of the ages and plot names of the legitimately complicated sites.

[This file](https://github.com/forc-db/ForC_private/blob/master/data%20to%20add/srdb/new_data/plot_names_changes_for_issue_183.csv) keeps a record of old vs new plotnames.

I used [this file](https://github.com/forc-db/ForC_private/blob/master/data%20to%20add/srdb/new_data/how_to_fix_stuff_in_issue_183.xlsx) to write down how things needed to be changed and as you can see there are a handfull of sites that need more investigation. I'll try to get to them soon.

I'll look into the cases where age = NAC

**Comment by ValentineHerr** [comment]
update on issue #181
I made some progress on some of the "age=NAC" issues but there is still a few that need investigation (see [this file](https://github.com/forc-db/ForC_private/blob/master/data%20to%20add/srdb/new_data/how_to_fix_stuff_in_issue_183_age_NAC.xlsx))
We are limited by online access of papers.
Also, in a lot of cases there is only one paper referenced for the site, so all ages (including age = NAC) come from the same paper, which makes me think that that paper must be mentioning a  site of unknown age.

**Comment by teixeirak** [comment]
> There are a few instances where the species listed in veg.notes don't match with the dominant.veg classification. The description in the plot name often does match the dominant.veg classification, so we would have to decide if we want to override this or not @teixeirak @ValentineHerr . A running list of IDs in ForC to double check is:

@beckybanbury , I've checked all of these and updated dominant.veg and plot names. We will need to rerun the plots script (+ others), but I'm still working on updates so won't do it now.

**Comment by teixeirak** [comment]
@ValentineHerr, I'm just realizing that I should have recorded plot name changes in [this file](https://github.com/forc-db/ForC_private/blob/master/data%20to%20add/srdb/new_data/plot_names_changes_for_issue_183.csv). I have to go to a meeting now and probably won't have time to go back and do that today. My commits in the past hour or so included plot name changes.

**Comment by teixeirak** [comment]
I have made some changes that will require re-running the duplicates code, ForC simplified, etc. There are still more manual fixes to be made, so probably no point in doing that now.

**Comment by teixeirak** [comment]
List of specific fixes needed:
- [ ] records 20464-20911 - citation ID may be wrong for non-respiration variables (see `source.notes` for citations, which are listed completely in SRDB references data set)
